src/components/Country.jsx

```diff
@@ -49,7 +49,7 @@ const Country = () => {
             <li className={styles.countryInformationList}>
               {' '}
               Start of Week:
-              {country.startOfWeek}
+              {country.week}
             </li>
           </ul>
         </div>
```

Comment:
Please make sure that your implemented design follows the provided design guidelines. 

- [ ] For your main page please make sure that there is no white space around and your design is responsive. Also the listed countries in the column should have alternative color not the same colors for example if first card is `#5788e5` the second one should be `#4369b2`

Provided design|Your design
---|---
<img width="363" alt="Screenshot 2023-08-05 at 2 47 27 PM" src="https://github.com/Kidd254/Metrics-WebApp-React-Redux/assets/44798044/3f1af490-cda4-47e3-b477-8f9cf69126e1">|<img width="363" alt="Screenshot 2023-08-05 at 2 47 47 PM" src="https://github.com/Kidd254/Metrics-WebApp-React-Redux/assets/44798044/93608d47-6c93-4fdc-8be4-e2f2cee46492">

- [ ] Please also make sure that the design of the detailed page also follows the provided guidelines. See the screenshots for comparison 

Provided design|Your design
---|---
<img width="276" alt="Screenshot 2023-08-05 at 2 50 34 PM" src="https://github.com/Kidd254/Metrics-WebApp-React-Redux/assets/44798044/ae8608a3-dce7-460c-ad11-48826c8607fe">|<img width="276" alt="Screenshot 2023-08-05 at 2 31 17 PM" src="https://github.com/Kidd254/Metrics-WebApp-React-Redux/assets/44798044/638ce0ab-fb09-4c95-af8f-7b57bec94fe7">